lib/msf/ui/console/command_dispatcher/developer.rb

```diff
@@ -217,6 +220,18 @@ def cmd_reload_lib_tabs(str, words)
     tab_complete_filenames(str, words)
   end
 
+  def cmd_reload_diff_files
```

Comment:
Since this is not a command to be dispatched, it should not have the the `cmd_` prefix and should be closer to the `reload_file` method, since they have similar functionality.